engine/class_modules/sc_rogue.cpp

```diff
@@ -10675,6 +10886,22 @@ void rogue_t::create_buffs()
   if ( spell.sepsis_buff->ok() )
     buffs.sepsis->set_initial_stack( as<int>( talent.shared.sepsis->effectN( 6 ).base_value() ) );
 
+  // Hero
+  // Fatebound
+  buffs.fatebound_coin_heads = make_buff<damage_buff_t>( this, "fatebound_coin_heads_buff", spell.fatebound_coin_heads_buff );
+  buffs.fatebound_coin_heads->set_stack_change_callback( [this]( buff_t*, int, int new_stacks ) {
+      if ( new_stacks == 7 )
+        buffs.fatebound_lucky_coin->trigger();
+    } );
+  buffs.fatebound_coin_tails = make_buff( this, "fatebound_coin_tails_buff", spell.fatebound_coin_tails_buff )
+    ->set_stack_change_callback( [this]( buff_t*, int, int new_stacks ) {
+      if ( new_stacks == 7 )
+        buffs.fatebound_lucky_coin->trigger();
+    } );
+  buffs.fatebound_lucky_coin = make_buff( this, "fatebound_lucky_coin_buff", spell.fatebound_lucky_coin_buff )
+    ->set_pct_buff_type( STAT_PCT_BUFF_AGILITY )
+    ->set_default_value_from_effect_type( A_MOD_TOTAL_STAT_PERCENTAGE ); // TODO: Add omni-stat buff type
```

Comment:
Definitely need some direction here - this buff has effects for buffing like 9 different stats by 1%. (agi, stam, crit, vers, mastery, attack speed (? haste?), leech, avoidance, movement speed). I feel like the automatic buff parser *should* be able to handle it, but... I don't know if it does? The summary doesn't display any affected stats on the buff details, anyway, even with the above two lines trying to at least force an agi buff. 🫤

Reply:
I'd recently talked to Gas about this, actually.

You can invoke multiple `set_pct_buff_type`'s on a single buff, assuming they all provide the same value. If they don't provide the same value, you're out of luck.

It will end out registering the invalidation for each of the mapped `pct_buff_type` caches and add it to the matching buff vector for that `pct_buff_type`, as seen here:

https://github.com/simulationcraft/simc/blob/dbea3b61ccbdd95bf9e8ac82409872eccb19cd05/engine/buff/buff.cpp#L1143-L1154

Reply:
Hm, I guess that works here since all the buffs are 1% - but there's no PCT_BUFF enum members for the 3 tertiary stats. Should I work on adding them? Or is there a different method for handling movement speed/avoidance/leech buffs?

Reply:
I personally wouldn't bother, as many of those effects aren't [entirely] implemented and don't have an impact on any results. You could if you'd like, though.